### ISSUE TITLE
Locale language setting in Excel different from English

### ISSUE THREAD

**Issue by pb20389** [OECDSTD/sdmx-matrix-generator#53]
When the Language Setting for Excel is e.g. "German", the Boolean values "True", "False" for "isFinal" are automatically converted to "Wahr" or "Falsch". 

When importing the SDMX-XML-File, generated by the SDMX-Matrix-Excel-Generator, into the SDMX-Registry, an error is generated that the XML file is not valid because of the wrong value in "isFinal".

Thank you
Markus P.

**Comment by dbarr99** [comment]
Hello, thanks for raising this issue. I tested it with the Hungarian language and resolved a bug in the VBA. Please test it with the new version and reraise the issue if it's still a problem.

**Comment by pb20389** [comment]
Good afternoon,

thanks for the adjustments, 

I just tested the fixes for the "isFinal" value, but it does not quite work for me yet. 

There are still some small VBA-Code corrections to be made regarding the "isFinal", 
see the Word document in the attached Zip-File.

Then it should work.

See also the corrected Excel file which I included in the Zip-File.

Thanks a lot

Markus Priller

[Corrected_Test_File_and_Instructions.zip](https://github.com/OECDSTD/sdmx-matrix-generator/files/4884686/Corrected_Test_File_and_Instructions.zip)
